Mathlib/Analysis/Analytic/Meromorphic.lean

```diff
@@ -310,6 +327,53 @@ lemma id {U : Set 𝕜} : MeromorphicOn id U := fun x _ ↦ .id x
 lemma const (e : E) {U : Set 𝕜} : MeromorphicOn (fun _ ↦ e) U :=
   fun x _ ↦ .const e x
 
+/-- The set where a meromorphic function has infinite order is clopen in its domain of meromorphy.
+-/
+theorem clopen_of_order_eq_top {U : Set 𝕜} [CompleteSpace E] (h₁f : MeromorphicOn f U) :
```

Comment:
```suggestion
theorem isClopen_setOf_order_eq_top {U : Set 𝕜} [CompleteSpace E] (h₁f : MeromorphicOn f U) :
```
Do we have this theorem for holomorphic functions, by the way? It would be strange to have it in the meromorphic case but not the holomorphic casr...